src/pages/console/uibuilder/responsive.mdx

```diff
@@ -22,39 +24,89 @@ By default, all Amplify UI provides the following [breakpoints as part of the Am
 }
 ```
 
-Configure your Figma components with variants with names that match the breakpoint values. In the example below, our `TopBar` component has the following variants:
-- `variation: default`
-- `variation: small`
+Breakpoints are set at the [Amplify UI Theme](https://ui.docs.amplify.aws/react/theming/responsive) level, which means they apply to all components in your Figma file.  You can customize these breakpoints using the Amplify Theme Editor plugin:
+1. Right click in your Figma project and select "Plugins" > "AWS Amplify Theme Editor"
```

Comment:
I think you need an empty line above here for the docs to interpret this correctly. (not sure if this is still the case)